src/featureFlags/useFlag.ts

```diff
@@ -0,0 +1,14 @@
+import { Name as FlagName } from "./availableFlags";
+import { useFeatureFlagContext } from "./context";
+import { FlagContent } from "./FlagContent";
+
+export const useFlag = (name: FlagName) => {
```

Comment:
I'd be good if you provide example of testing flags, maybe some simple Wrapper with static strategies list provided?